packages/graphiql/example/index.html

```diff
@@ -107,9 +107,12 @@
       // use fetch, and could instead implement graphQLFetcher however you like,
       // as long as it returns a Promise or Observable.
       function graphQLFetcher(graphQLParams) {
-        // This example expects a GraphQL server at the path /graphql.
+        // When working locally, the example expects a GraphQL server at the path /graphql.
+        // In a PR preview, it connects to the Star Wars API externally.
         // Change this to point wherever you host your GraphQL server.
-        return fetch('/graphql', {
+        const isDev = window.location.hostname === 'localhost'
```

Comment:
I normally alias my `localhost:1234` addresses to e.g. `graphiql.meh` using [mehserve](https://github.com/benjie/mehserve) (and I don't think I'm the only person who does similar, there was a lot of anger when `.dev` was registered by Google!). Can we use an allow-list approach rather than a block-list approach? Is there a regexp we can use against the domain that'll recognise netlify?

```suggestion
        const isDev = !window.location.hostname.endsWith('.netlify.com')
```

Reply:
agreed. as in my PR review as well, we need to extract this example and/or somehow leverage swapi-graphql.

Reply:
this hostname may not end with `.netlify.com` - for example, we might have `graphql.org/swapi` be a consumable GraphQL Service soon

Reply:
`/(^|\.)netlify\.com$|(^|\.)graphql\.org$/`? We can always update it later, I assume?